contracts/adapters/stake-dao/StakeDAOTokenAdapter.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: None
+
+pragma solidity 0.6.5;
+pragma experimental ABIEncoderV2;
+
+import {ERC20} from "../../ERC20.sol";
+import {TokenMetadata, Component} from "../../Structs.sol";
+import {TokenAdapter} from "../TokenAdapter.sol";
+
+interface Vault {
+    function token() external view returns (address);
+
+    function getPricePerFullShare() external view returns (uint256);
+}
+
+/**
+ * @title Token adapter for Stake DAO Vaults.
+ * @dev Implementation of TokenAdapter abstract contract.
+ * @author Elephant memory/strength
+ */
+contract StakeDaoTokenAdapter is TokenAdapter {
+    address public sdveCrv = 0x478bBC744811eE8310B461514BDc29D03739084D;
+
+    function setSdveCrv(address newSdveCrv) external {
+        sdveCrv = newSdveCrv;
+    }
```

Comment:
is there a reason it may be useful?

Reply:
yes, so that you can update the sdveCrv address when it's changed in the future, without redeploying StakeDaoTokenAdapter.sol

Reply:
okay, but I will break the integration anyway
so I think there are 2 things you should do
- inherit from Ownable contract from this repo and use `onlyOwner` here
- notify us about change of `sdveCrv` address, so we change it here and in our backend